package.json

```diff
@@ -13,11 +13,9 @@
 		"node": ">=10"
 	},
 	"scripts": {
-		"build": "rollup index.js --format=umd --name=ky --file=umd.js",
-		"prepublishOnly": "npm run build",
-		"test": "xo && nyc ava && tsd-check && npm run cypress:run",
-		"cypress:open": "cypress open",
-		"cypress:run": "cypress run"
+		"pretest": "rollup index.js --format=umd --name=ky --file=umd.js",
```

Comment:
We can avoid this by using `page.addScriptTag({path: './index.js', type: 'module'})`. Is there a reason you didn't want to do that? Not a big deal, but loading the source directly feels a little more clean to me and it will speed up the tests a bit.

Reply:
Why did you rename this?

Reply:
@sholladay Yea it would be nice if I could do that but I don't think it's practical because if you use a module script you have to import the module in the `evaluate` code right? But then if you have an import statement outside of the top level you get a parse error. I can think of a couple ways around this but they're not great.

@szmarczak Rename `build` to `pretest`? Out of habit initially, but really because the browser tests test against the UMD build so it's convenient to use the `pretest` npm convention since it automatically gets run before `test` runs without needing to explicitly run it. You can still run it manually (as is the case with prepublishOnly) if you want though.

Reply:
> if you use a module script you have to import the module in the `evaluate` code right?

Yeah, I guess you're right. Just didn't think it all the way through. We could make a helper for that, but maybe it's not worth it at that point.

Reply:
> but really because the browser tests test against the UMD build so it's convenient to use the `pretest` npm convention since it automatically gets run before `test` runs without needing to explicitly run it. You can still run it manually (as is the case with prepublishOnly) if you want though.

I prefer being explicit. It's weird having to run `npm run pretest` to build. The script should be called `build`.

Reply:
> I can think of a couple ways around this but they're not great.

Just curious, what are the ways around it? Using `import` would be much nicer, so would be good to know the downsides.

Reply:
I will take care of putting the `build` script back. I'm working on some test tweaks right now.

Reply:
@sindresorhus 
1. Pass a string to `page.evaluate` instead of a function. No parse error but writing JavaScript in a string would be annoying. I could see it being a viable option in the future though if there was a tagged template literal for JavaScript that editors could syntax highlight.
2. Extract the evaluation code to another file. This creates more files, essentially takes some of the test code out of the test file, and changes the evaluation interface a bit. I actually just switched over to this method personally despite its downsides. I updated [`puppeteer-helper`](https://github.com/ianwalter/puppeteer-helper) to make this easy, you just call `t.evaluate()` with a relative path to the evaluation script.

Reply:
Okey. Makes sense. Thanks for elaborating.